app/controllers/boards_controller.rb

```diff
@@ -36,7 +35,7 @@ def create
 
     respond_to do |format|
       if @board.save
-        format.html { redirect_to ‘/pins/new’, notice: 'Board was successfully created.' }
+        format.html { redirect_to '/pins/new', notice: 'Board was successfully created.' }
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.